### PR TITLE
Add docs for datadog `uri` property configuration

### PR DESCRIPTION
**Pull request by r1jsheth**
Closes https://github.com/micrometer-metrics/micrometer-docs/issues/277

### CODE REVIEW

File: src/docs/implementations/datadog.adoc [hunk 1]
```diff
@@ -71,6 +71,18 @@ You can set the tag by using common tags, as follows:
 registry.config().commonTags("host", "my-host");
 ----
 
+`management.metrics.export.datadog.uri` is an important property to configure.
```
Comment (shakuzen) [inline src/docs/implementations/datadog.adoc:74]: `management.metrics.export.datadog.uri` is specific to Spring Boot, while we try to be more generic in the Micrometer documentation. What do you think about updating this to just refer to the `uri` config method on `DatadogConfig`? Note [the Spring Boot documentation](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#actuator.metrics.export.datadog) already has mention of configuring the uri if not using the default site.

Comment (r1jsheth) [reply src/docs/implementations/datadog.adoc:74]: You're right. I will update

Comment (r1jsheth) [reply src/docs/implementations/datadog.adoc]: Updated

**Review by shakuzen — APPROVED** [review]
Thank you.

**Pull request merged**